src/documentation/common.md

```diff
@@ -0,0 +1,33 @@
+## CO2 emissions and fallback modes
+Physical modes may not contain CO2 emissions. If the value is missing, we are
+using default values (see below), mostly based on what is provided by
+[ADEME](https://www.ademe.fr).
+
+Physical Mode     | CO2 emission
```

Comment:
maybe considering adding the unit `gCO2eq` (best writing I found)

Reply:
I looked a little bit and I also found sometimes a dash `-` to separate the `eq`. I could propose something like **gCO<sub>2</sub>-eq** (I found out that also putting the `2` as an indice might help too).

Reply:
> I would propose gCO2-eq/km, to make it more explicit that it's grams of CO2 per km?

Thanks to @papailio, here is the final form, done in fc5c4ab224ae5190f3e898423c7e49570d272aa9.